CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - pluralize thumbnail url
 - Simplify template to frontend communication by using JSON instead of multiple data-attributes
 - Rename all is_ready_to_* model properties to is_ready_to_show
+- Change filename when a user download a resource. It now set with the playlist title
```

Comment:
Change filename when a user downloads a resource. It now uses the playlist title